.shippable.yml

```diff
@@ -74,6 +74,8 @@ build:
       - rm test_file.txt
       - ccache -s
     on_failure:
+      - echo "Dumping the capability database in case we are affected by #9992"
+      - cat $HOME/.cache/zephyr/ToolchainCapabilityDatabase.cmake
```

Comment:
actually, you want to check if the file exists first, we might have failed before the cache was created

Reply:
Good catch, fixed.